xarray/tests/test_sparse.py

```diff
@@ -856,6 +856,10 @@ def test_dask_token():
     import dask
 
     s = sparse.COO.from_numpy(np.array([0, 0, 1, 2]))
+
```

Comment:
xfail this instead?

Reply:
Yeah that was an option. But it would have completely disabled the test, and the test is really about not accidentally invoking ``self.values`` (which in turn invoked ``self._variable._data.__array__()``) on NEP18-compatible backends. It is not really about testing sparse specifically; it just conveniently relies on the fact that ``COO.__array__`` raises an exception. A more correct, but also more laborious, test could have created a NEP18-compatible dummy class on the fly.